config/locales/spotlight.en.yml

```diff
@@ -261,6 +261,12 @@ en:
           non_repo_item: "Add non-repository item"
       new:
         header: Import items
+    invitation_mailer:
+      invitation_instructions:
+        hello: "Hello %{email}!"
+        someone_invited_you: "The Exhibits Administrator has invited to be a member of '%{exhibit_name}' at %{url}. You can accept it through the link below.</p>"
```

Comment:
Actually, I don't think this particular key should have HTML.  The `<p>` is in the mailer body (and this is actually causing a duplication `</p>`.  I'll remove the html and add a `_html` to the keys that do include markup.